scripts/lamp

```diff
@@ -1,7 +1,28 @@
 #!/usr/bin/env bash
 # Because genies come from lamps.
 
+set -eou pipefail
+
+DIR=$(pwd)
+
 # for mongodbtoolchain python3
 export PATH=/opt/mongodbtoolchain/v3/bin:$PATH
 
+if [[ "$*" != *-g* ]]
```

Comment:
this syntax is nice but mystical - suggest a comment `# Check if called with -g flag`